ad-insertion/video-analytics-service/feeder/main.py

```diff
@@ -114,10 +124,11 @@ def process_stream(streamstring):
                 status, fps = get_analytic_status(instanceid.strip(), pipeline)
                 print("VA feeder: segment status : " + status, flush=True)
                 if status == 'COMPLETED':
-                    print("VA feeder: Avg fps: " + str(fps), flush=True)
+                    send_video_analytics_fps(fps)
```

Comment:
can we add debug log with the status,fps - was useful to have

Reply:
for every send kafka message, they have print statement in message.py.